recipes/genbooster/meta.yaml

```
@@ -0,0 +1,83 @@
+{% set name = "genbooster" %}
+{% set version = "0.5.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/Techtonique/{{ name }}/archive/v{{ version }}.tar.gz
+  sha256: cd1582ea4064f39041fc47771effad2b0232118c065ecf8f99a87b41d6083023
+
+build:
+  number: 0
+  script: "{{ PYTHON }} setup.py build_ext --inplace && {{ PYTHON }} -m pip install . -vv"
```

You should not need to build the extension first.

```suggestion
  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
```